experiments/integrated/ozark/ozark_met_drivers_FLUXNET.jl

```diff
@@ -121,24 +126,23 @@ lat = FT(38.7441) # degree
 long = FT(-92.2000) # degree
 
 function zenith_angle(
-    t::FT,
+    t,
     orbital_data,
     ref_time;
     latitude = lat,
     longitude = long,
     insol_params = earth_param_set.insol_params,
-) where {FT}
+)
     # This should be time in UTC
     dt = ref_time + Dates.Second(round(t))
-    FT(
-        instantaneous_zenith_angle(
-            dt,
-            orbital_data,
-            longitude,
-            latitude,
-            insol_params,
-        )[1],
-    )
+    # TODO do we need to wrap this in FT? can we get FT from a param other than t?
```

Comment:
can we pass FT as a keyword argument after it is defined (closure)?

it may be stored in the orbital data also

Reply:
I don't think I understand - how would we use a closure to get FT?

Reply:
hm, I guess I thought this would be a closure
```
FT = Float64
function foo(a, b; FT = FT)
 # use FT
end
```
and then pass foo into the model. But, maybe that isnt a closure. 

I think we can get FT from the insol params too :)

Reply:
This works:
```
function zenith_angle(
    t,
    orbital_data,
    ref_time;
    latitude = lat,
    longitude = long,
    insol_params::Insolation.Parameters.InsolationParameters{FT} = earth_param_set.insol_params,
) where {FT}
    # This should be time in UTC
    dt = ref_time + Dates.Second(round(t))
    # Orbital Data uses Float64, so we need to convert to our sim
    # FT.
    FT(
        instantaneous_zenith_angle(
            dt,
            orbital_data,
            longitude,
            latitude,
            insol_params,
        )[1],
    )
end

```

Ill change to this for the Ozark and Vaira site runs (PR adding the Vaira site), we can change next week when youre back if you prefer a different approach!

Reply:
Looks good, thank you!